lmdeploy/model.py

```diff
@@ -1299,6 +1299,23 @@ def match(cls, model_path: str) -> Optional[str]:
             return 'internvl-zh-hermes2'
 
 
+@MODELS.register_module(name='starcoder2')
```

Comment:
If it is not a chat model or an instruct model, there is no need to add a template. 
`best_match_model` will fallback to base template